scripts/genproto.sh

```diff
@@ -114,10 +114,10 @@ schwag -input=Documentation/dev-guide/apispec/swagger/rpc.swagger.json
 # install protodoc
 # go get -v -u github.com/coreos/protodoc
 #
-# by default, do not run this option.
-# only run when './scripts/genproto.sh -g'
+# run './scripts/genproto.sh skip-protodoc'
+# to skip protodoc generation
 #
-if [ "$1" = "-g" ]; then
+if [ "$1" != "skip-protodoc" ]; then
```

Comment:
@gyuho I agree with this I have never ran the script without the intention of protodoc gen.